odata-mock/README.md

```diff
@@ -5,155 +5,31 @@
 # OData Mock
 [![npm version](https://badge.fury.io/js/varkes-odata-mock.svg)](https://badge.fury.io/js/varkes-odata-mock)
 
-Mocks rest calls given an OData specification
+The OData mock mocks the application APIs based on the OData specification.
 
-After startup, you have access to:
-- odata API - http://localhost:10000/odata/
-- odata metadata - http://localhost:10000/odata/$metadata
-- rest API - http://localhost:10000/api/console
+## Features
 
-## Starting the application
+The OData mock brings you the following features:
 
-### Run local
+- Supports OData edmx specification: Copy the edmx specification to the [EDMX_MODEL_SPECIFICATION.xml](https://github.com/kyma-incubator/varkes/blob/master/examples/odata-mock-app/EDMX_MODEL_SPECIFICATION.xml) file and the Engine will automatically create data model files for the entities specified in the specification
 
-To run it local, specify a config file (as located in the /test folder) like that:
-```
-npm install
-node server/server <varkes_config.js>
-```
+- Records every Request made to the node: Uses the [morgan](https://www.npmjs.com/package/morgan) logging framework to create `requests.log` file. This file contains the called URLs, the header of the request and the body of the request if exists.
 
-### Run local using docker
+- Returns the OData specification as metadata: Call `/odata/$metadata` to see the OData edmx specification being use in xml format
 
-To run it using docker, call:
-```
-docker run -p 10000:10000 eu.gcr.io/kyma-project/incubator/develop/varkes-odata-mock:latest
-```
+- Based on the n-odata package: The odata-mock application is based on the [n-odata-server](https://github.com/htammen/n-odata-server) project, which is based on [loopback](https://loopback.io/).
 
-### Run in Kyma
+After startup, you have access to:
+- odata API - `http://localhost:10000/odata/`
+- odata metadata - `http://localhost:10000/odata/$metadata`
+- rest API - `http://localhost:10000/api/console`
 
-To run the mock using Kyma as runtime envrironment, run the following kubectl command to set up a namespace:
 
-```bash
-kubectl create namespace mocks
-kubectl label namespace mocks env=true
-```
+## Run mock
 
-and to deploy the mock
-```bash
-kubectl apply -n mocks -f https://raw.githubusercontent.com/kyma-incubator/varkes/master/odata-mock/deployment/deployment.yaml
+To run the mock locally, specify the [varkes config](https://github.com/kyma-incubator/varkes/blob/master/odata-mock/src/test/varkes_config.json) file:
```

Comment:
See the instructions I gave for app-connector-client. It will be exactly the same here. No mentioning of varkes config at all for now

Reply:
I will change it for all three docs.